fossa/info/action.yml

```diff
@@ -67,3 +79,38 @@ runs:
         echo "head-revision=${HEAD_REVISION}"
       } >> $GITHUB_OUTPUT
     shell: bash
+  - name: Get additional PR info
+    if: ${{ steps.info.outputs.is-pull-request == 'true' }}
+    id: info-pr
+    env:
+      BASE_REF: ${{ steps.info.outputs.base-ref }}
+      BASE_REVISION: ${{ steps.info.outputs.base-revision }}
+      GH_TOKEN: ${{ inputs.github-token }}
+    run: |
+      # Check recent commits of the base branch for existing scanning results
+      # history depth to search for scanning results
+      depth=5
+      # Ensure history is available for the base branch up to the specified depth
+      git fetch origin "${BASE_REF}" --depth="$depth"
+      # Check if the base branch or its recent parents have scanning results
+      base_ref_with_scanning_results=""
+      for i in $(seq 0 $((depth-1))); do
+        commit=$(git rev-parse "${BASE_REVISION}~$i" 2>/dev/null)
+        # Check if a workflow run of "Check Licenses" succeeded for the commit
+        if gh run list --commit "$commit" --json workflowName,conclusion \
+          --jq '[.[] | select(.workflowName == "Check Licenses" and .conclusion == "success")] | any' | grep -q true; then
+          if [ "$i" -gt 0 ]; then
+             echo "Scanning result found for commit=$commit, which is $i commit(s) behind the base branch/ref."
```

Comment:
do you think that's worth annotating in the github job for visibility?

Reply:
I don't think we want to advertise this outside of the logs for now, as this is no actionable for end-users (but noise).